src/skin/Theme.js

```diff
@@ -65,18 +65,17 @@ Theme.prototype.setSkin = function(comp, id, skin) {
 /**
  * Set up the asset loader and load files
  *
- * @method loadImage
+ * @method addImage
  * @param jsonPath {Array}
  */
-Theme.prototype.loadImage = function(jsonPath) {
+Theme.prototype.addImage = function(jsonPath) {
```

Comment:
I changed the name because this function now only adds the file to the loader and does not load it. You need to call load on the GOWN.loader/PIXI.loader manually after you added all your files (this changed in some PIXI 4-version ).